tests/Meilisearch.Tests/SettingsTests.cs

```diff
@@ -438,6 +442,49 @@ public async Task ResetTypoTolerance()
             await AssertGetEquality(_index.GetTypoToleranceAsync, _defaultSettings.TypoTolerance);
         }
 
+        [Fact]
+        public async Task GetPagination()
+        {
+            await AssertGetEquality(_index.GetPaginationAsync, _defaultSettings.Pagination);
+        }
+
+        [Fact]
+        public async Task UpdatePagination()
+        {
+            var newPagination = new Pagination
+            {
+                MaxTotalHits = 20
+            };
+
+            var returnedPagination = new Pagination
+            {
+                MaxTotalHits = 20
+            };
+
+            await AssertUpdateSuccess(_index.UpdatePaginationAsync, newPagination);
+            await AssertGetEquality(_index.GetPaginationAsync, returnedPagination);
+        }
+
+        [Fact]
+        public async Task ResetPagination()
+        {
+            var newPagination = new Pagination
+            {
+                MaxTotalHits = 30
+            };
+
+            var returnedPagination = new Pagination
+            {
+                MaxTotalHits = 30
+            };
+
+            await AssertUpdateSuccess(_index.UpdatePaginationAsync, newPagination);
+            await AssertGetEquality(_index.GetPaginationAsync, returnedPagination);
```

Comment:
Here too just used `newPagination` in the both tests?

Reply:
Fixed.